examples/basic/src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import { GMaps } from "react-gmaps";
+
+function App() {
+  return (
+    <div style={{ height: "100vh" }}>
+      <GMaps />
```

Comment:
The map doesn't seem to work until I add `loaderOptions={{ apiKey }}`.

Reply:
Interesting :thinking: 
What do you mean by "doesn't seem to work"? The map doesn't show up and you get a blank page?

On my end, the map renders but with the typical GoogleMaps warning "For development purpose only"
<details>
<summary>
Screen capture
</summary>
<img src="https://github.com/MicheleBertoli/react-gmaps/assets/35486736/f03f9447-d664-48e3-b88b-10960be54995" />
</details>

Reply:
Yeah, I see the same as your screenshot (which goes away with the additional prop).
Not a blocker, just a heads-up - especially considering that "basic" will likely be the most visited example.

Reply:
I'd definitely prefer not to have this warning all over the map :confused:
I think it goes away only because you filled in an API key in the `.env` file.

But i'll add the variables because it might make it a bit more clear why we have this warning :+1: